docs/development-environment.md

```diff
@@ -192,6 +190,50 @@ There are different types of builds:
 
 	Are pre-commit and pre-push hooks slowing down a major refactor or draft PR? Run `jetpack draft enable` to make them less aggressive (they will still run, but won't block for warnings), and `jetpack draft disable` when you're ready for them again.
 
+* ### Syncing local changes with Unison
+  
+  In some cases, you may need to test Jetpack (jetpack-mu-wpcom, in particular) changes by syncing your changes to another machine (rather than using Docker). This outlines a strategy for syncing changes in real-time using the [Unison](https://github.com/bcpierce00/unison) file sync tool combines with the [unison-fsmonitor](https://github.com/benesch/unison-fsmonitor).
+
+  This approach may be especially useful for Automatticians who are testing changes on their WPCOM sandbox. Using Unison can be more efficient than rsync (or `jetpack rsync`).
+
+  #### Installing Unison
+
+  Please see the respective Unison and usion-fsmonitor repositories for full installation instructions. 
+
+  On OSX, you can use [Homebrew](https://brew.sh/) to quickly install both tools:
+
+  - `brew install unison`
+  - `brew install autozimu/formulas/unison-fsmonitor`
+
+  #### Configuring Unison
+
+  Once Unison is installed, you'll want to create a preferences file. On OSX/Linux, that preferences file would be placed in `~/.unison` and could be called something like `jetpack-plugin-sync.prf`. See the Unison documentation for instructions for other platforms.
```

Comment:
Can we point users to docs via the `unison` command? It looks like there are some decent docs available by running the following commands (with the pipe through `less` to make it easier to scroll/page):
 * `unison -doc tutorial | less`
 * `unison -doc basics | less`
 * `unison -doc running | less`